lib/xdparser2.ts

```diff
@@ -283,33 +296,66 @@ function getLine(body: string, substr: string) {
 
 // This came from the original, I think it's pretty OK but maybe it could be a bit looser
 const clueRegex = /(^.\d*)\.\s(.*)\s\~\s(.*)/
+const clueMetaRegex = /(^.\d*)~(.*)\.\s(.*)/
```

Comment:
Agree, I'm sorta siding one `D25 ^Hint. The man who wrote marvel comics. Two generic male names.` where the whitespace is enforced, It feels a lot like the markdown syntax for reference footnotes 

```
Here is a simple footnote[^1].

[^1]: My reference.
```

Here is a simple footnote[^1].

I wonder if `D25 ^Hint: The man who wrote marvel comics. Two generic male names.` is not better because (w/ `:`) it truly breaks from the original comment, is closer to existing md syntax and is quite obviously not the main syntax for a line.


[^1]: My reference.